simulation/sub8_gazebo/worlds/a_whole_new.world

```diff
@@ -17,7 +17,7 @@
       <uri>model://sun</uri>
     </include>
 
-    <!-- Never delete this -->
+    <!-- Never delete this, uncomment to bring back the ghost cessna
```

Comment:
Just amended the pull request, that should fix everything save the YAML scripts which I am now attempting to implement.